$NetBSD: patch-ac,v 1.2 2012/10/09 17:53:51 marino Exp $

GCC 4.6+ chokes on "--warn-common" which older versions of GCC left fall to
the linker.  The missing "," after "-Wl" is no longer acceptable.
Rest is from original version of patch

--- configure.orig	2008-12-02 18:04:42.000000000 +0000
+++ configure
@@ -102,7 +102,7 @@ extralibs="-lm"
 bigendian="no"
 SHFLAGS=-shared
 need_inet_aton="no"
-LDFLAGS="$LDFLAGS -Wl --warn-common"
+LDFLAGS="$LDFLAGS -Wl,--warn-common"
 CFLAGS=""
 CPPFLAGS=""
 GPAC_SH_FLAGS=-lpthread
@@ -217,12 +217,16 @@ case "$cpu" in
   ;;
   x86_64|amd64)
     cpu="x86"
-    is_64="yes"
+    if test "$linux" = "yes" ; then
+      is_64="yes"
+    fi
     canon_arch="`cc -dumpmachine | sed -e 's,\([^-]*\)-.*,\1,'`"
     if [ x"$canon_arch" = x"x86_64" -o x"$canon_arch" = x"amd64" ]; then
       if [ -z "`echo $CFLAGS | grep -- -m32`"  ]; then
         cpu="x86_64"
-	libdir="lib64"
+        if test "$linux" = "yes" ; then
+	  libdir="lib64"
+	fi
         #that's a bit crude...
         PIC_CFLAGS="-fPIC -DPIC"
         want_pic="yes"
